releng/runtime/pom.xml

```diff
@@ -62,7 +62,7 @@
 				<repository>
 					<id>swtchart</id>
 					<layout>p2</layout>
-					<url>https://download.eclipse.org/swtchart/releases/0.12.0/repository</url>
+					<url>https://download.eclipse.org/swtchart/releases/0.14.0/repository</url>
```

Comment:
Runtime is supposed to support the oldest possible version. Will it be able to work with SWTChart 12?